docs/standards/universal-profile/02-lsp1-universal-receiver-delegate.md

```diff
@@ -3,12 +3,67 @@ sidebar_label: 'LSP1 - Universal Receiver Delegate'
+![ur-delegate-transaction](../../../static/img/ur-delegate-transaction.jpg)
+
+
+## Implementations
+
+There is several implementations of the standard, **[LSP1UniversalReceiverDelegateUP](../smart-contracts/lsp1-universal-receiver-delegate-up.md)** contract is one of them that is used as a delegate to the **universalReceiver** function of **UniversalProfile** contract. 
+
+At the moment this contract allows receiving and sending tokens & vaults and registers the keys representing them according to **[LSP5-ReceivedAssets](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-5-ReceivedAssets.md)** and **[LSP10-ReceivedVaults](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-10-ReceivedVaults.md)** Standards.
+
+### Token transfer scenario
+
+One of the **possible scenarios** is a token transfer between Alice and Bob. Alice wants to transfer a token owned by her UniversalProfile to the UniversalProfile of her friend Bob. 
+
+**1.** It call the **transfer** function on the token contract through the KeyManager.
+
+
+![token-transfer-1](../../../static/img/token-transfer-1.jpg)
+
+
+**2.** The transfer function on the token contract will directly **trigger a hook** that will call the universalReceiver function on both sender and recipient UniversalProfiles.
+
+
+![token-transfer-2](../../../static/img/token-transfer-2.jpg)
+
+
+**3.** If the **UniversalReceiverDelegate** contract is set then the **universalReceiver** function will call the **UniversalReceiverDelegate** contract and execute the logic customized in it.
+
+
+![token-transfer-3](../../../static/img/token-transfer-3.jpg)
+
+
+**4.** The **UniversalReceiverDelegate** of **the UniversalProfile** allows the transfer and set **[LSP5-ReceivedAssets](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-5-ReceivedAssets.md)** keys on both Profiles through the KeyManager.
+
+
+
+![token-transfer-4](../../../static/img/token-transfer-4.jpg)
+
+
+
 
-:::info Coming Soon
-:::
 
 ## References
 
-- [LUKSO Standards Proposals: LSP1 - Universal Receiver (Standard Specification, GitHub)](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-1-UniversalReceiver.md)
-- [LSP1 Universal Receiver: Solidity implementations (GitHub)](https://github.com/lukso-network/lsp-universalprofile-smart-contracts/tree/develop/contracts/LSP1UniversalReceiver)
+- **[LUKSO Standards Proposals: LSP1 - Universal Receiver (Standard Specification, GitHub)](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-1-UniversalReceiver.md)**
+- **[LSP1 Universal Receiver: Solidity implementations (GitHub)](https://github.com/lukso-network/lsp-universalprofile-smart-contracts/tree/develop/contracts/LSP1UniversalReceiver)**
```

Comment:
Let's be consistent with other pages - none of the References are in bold in any other page, let's not make the ones from this page in bold